test/integration/ink_editor_spec.mjs

```diff
@@ -1294,3 +1294,45 @@ describe("Should switch from an editor and mode to others by double clicking", (
     );
   });
 });
+
+describe("Ink must update its color", () => {
```

Comment:
Could you also add a test like this for freetext annotations so that part of the PR is also covered?